### PR TITLE
Enable and fix functional tests

### PR DESCRIPTION
**Pull request by tmacro**
I found a stray `.only` when writing tests for #879 causing all but a single functional test suite to be disabled. The changes grew a bit large so I've split it into a separate PR here.

### CODE REVIEW

File: tests/functional/v2/task/testCreateCheckpoint.js [hunk 1]
```diff
@@ -16,6 +16,33 @@ const checkpointLabelToLevel = {
     usr: 'users',
 };
 
+// async function fetchRecords(warp10, className, labels, params) {
```
Comment (miniscruff) [inline tests/functional/v2/task/testCreateCheckpoint.js]: large commented block

File: tests/functional/v2/task/testRepair.js [hunk 2]
```diff
@@ -113,16 +119,18 @@ describe('Test Repair', function () {
...
+            { node: prefix },
+            { end: getTs(1), count: -1 },
+            '@utapi/decodeRecord',
+        );
         assert.strictEqual(series.length, 3);
         assertResults(totals, series);
 
         await repairTask._execute(getTs(0));
-        results = await warp10.fetch({
+        const results = await warp10.fetch({
```
Comment (jonathan-gramain) [inline tests/functional/v2/task/testRepair.js]: not possible or no benefit in using `fetchRecords` for this one?

**Review by jonathan-gramain — APPROVED** [review]

File: tests/functional/v2/task/testRepair.js (hunk 2 above)
Comment (tmacro) [reply tests/functional/v2/task/testRepair.js]: I didn't convert it as we're only concerned about the length and don't need to decode, but I'll change it for consistency

**Review by anurag4DSB — APPROVED** [review]

**Pull request merged**